### DISCUSSION TITLE
NAS / Shared Data

### DISCUSSION THREAD

**Discussion by firstchanceongit** [rern/rAudio#740] [Fixed Issues]
After removing a NAS and trying to add it as source again, as box "Shared Data" opens upon pushing the Plus-Button. Some data is pre-filled. It is not possible to save the NFS related information and thereby add the NAS as source. When hitting OK, the dialog responds after a short moment that the IP "xxx" is not found. "xxx" is the "Name" I had assigned the the NAS in the previous dialog (my understanding is that this is internal naming only).

It might be that I applied the most recent update fom 7 January 2022 (which seems to be related to "Shared Data") after I had initially added the NAS. I don't remember this exactly. Just mentioning it in case the order of steps may have had some influence ...

**Comment by ceejayemm** [comment]
@firstchanceongit 

I had this problem, see #734 It seems to be fixed with the Update branch version (from Addons, select and hold Update button until the UPDATE branch is displayed, then release and proceed with system update).

Chris

**Comment by srt9969** [comment]
Amazing! Was just about to post about this (I'm new to rAudio), as I've been fighting with it for the last couple hours.
Thank you!

**Comment by firstchanceongit** [comment]
Thanks - yes, that seems to have solved it for me. (Dialog box - now the correct one - comes also partly pre-filled, but that may be a feature and not a bug ...).
